pb/messages.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package pb;
+
+import "google/protobuf/timestamp.proto";
+
+// Status messages send generic meta information about the state of the inbox.
+message Status {}
```

Comment:
I haven't fully thought this through yet, but I want to make sure we can send generic status information both from the client to the server and from the server back to the client. We might have to make this fairly generic because there are a lot of things that could be included here, for example:

- geographic information for replication/data placement
- state of device pointers for synchronization 
- timestamps of last checked/cleaned etc 
- version update information